tests/command_test.py

```diff
@@ -203,6 +203,68 @@ def test_call_test_float(self):
             }],
         )
 
+    def test_call_test_special_judge(self):
+        def assert_line_count(expected: int) -> str:
+            return '{} -c "import sys; assert len(sys.stdin.readlines()) == {}"'.format(sys.executable, expected)
```

Comment:
```suggestion
            return '{} -c "import sys; assert len(sys.stdin.buffer.readlines()) == {}"'.format(sys.executable, expected)
```

Windows での動作を考えると、`str` (Unicode文字列) でなく `bytes` (バイト列) を使っておくとのが安全です。

Reply:
AppVeyor による Windows 上のテストで改行の問題が発生するやつは、以前に苦しめられたときのログ https://github.com/kmyk/online-judge-tools/pull/541#issuecomment-538003745 が役に立つかもしれません 。やばそうなら私や @fukatani に任せてもらってもよいです (手元に環境がないと不可能なので)